bitrise/publish-internal-podspec.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## I. Check parameter
+if [ -z $1 ] || ( [ "$1" != "internal" ] && [ "$1" != "external" ] ); then
+  echo "Invalid parameter.";
+  echo "  Usage: $0 {internal|external}";
+  exit 1;
+fi
+
+## II. Get publish version for information
+publish_version="$(grep "VERSION_STRING" $BITRISE_SOURCE_DIR/$VERSION_FILE | head -1 | awk -F "[= ]" '{print $4}')"
+echo "Publishing podspec for version" $publish_version
+
+if [ "$1" == "internal" ]; then
+
+  ## 1. Get path of internal podspec local repo
+  local_repo_path="$(pod repo | grep "$GIT_SPEC_REPO_NAME" | grep Path | head -1 | awk -F ": " '{print $2}')"
+
+  ## 2. Update podspec to the internal podspec local repo
+  resp="$(pod repo push $GIT_SPEC_REPO_NAME $BITRISE_SOURCE_DIR/$PODSPEC_FILENAME --allow-warnings)"
```

Comment:
Yes, that is a bad behavior. We have to fix this warning to eliminate that flag.
`[!] The validator for Swift projects uses Swift 3.0 by default, if you are using a different version of swift you can use a `.swift-version` file to set the version for your Pod. For example to use Swift 2.3, run:`